### PR TITLE
chore(ui): bump web console version

### PR DESCRIPTION
**Pull request by glasstiger**
### Fixed

- UX improvements around editor [#338](https://github.com/questdb/ui/pull/338)
- Running query is not cancelled when Run button is clicked

### Changed

- Support for notice responses from server, such as empty SQL notice [#334](https://github.com/questdb/ui/pull/334)

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
   ~
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```
Comment (bluestreak01) [inline core/pom.xml:25]: why was this changed?

Comment (glasstiger) [reply core/pom.xml:25]: intellij always changes it for me, something to do with formatting.
we have it setup differently somehow.
i think we share only java formatting settings.

**Review by bluestreak01 — APPROVED** [review]
